css/syntaxes.json

```diff
@@ -735,7 +738,7 @@
     "syntax": "sin( <calc-sum> )"
   },
   "single-animation": {
-    "syntax": "<time> || <easing-function> || <time> || <single-animation-iteration-count> || <single-animation-direction> || <single-animation-fill-mode> || <single-animation-play-state> || [ none | <keyframes-name> ]"
+    "syntax": "<time> || <easing-function> || <time> || <single-animation-iteration-count> || <single-animation-direction> || <single-animation-fill-mode> || <single-animation-play-state> || [ none | <keyframes-name> ] || <single-animation-timeline>"
```

Comment:
FYI `<time>` here is referred to as `<animation-duration>` [in the spec](https://drafts.csswg.org/css-animations-2/#typedef-single-animation), not sure if that's intentional that we use `<time>` here:

```ebnf
<'animation-duration'> || <easing-function> || <'animation-delay'> || <single-animation-iteration-count> || <single-animation-direction> || <single-animation-fill-mode> || <single-animation-play-state> || [ none | <keyframes-name> ] || <single-animation-timeline>
```

Reply:
And a couple of other differences ☝🏻 - am I looking at the right place in the spec?

Reply:
I've updated it in the next commit; it has probably not been updated since the days of CSS animations level 1.

Reply:
Looks good tnx